examples/storage/s3/incremental_learning/README.md

```diff
@@ -64,13 +68,42 @@ metadata:
 spec:
   url: "s3://kubeedge/data/helmet_detection/train_data/train_data.txt"
   format: "txt"
-  nodeName: $CLOUD_NODE
+  nodeName: cloud-node1
   credentialName: mysecret
 EOF
+```
 
+### Prepare Image
+This example uses the image:  
+
+```shell
+kubeedge/sedna-example-incremental-learning-helmet-detection:v0.3.1
 ```
-* Attach the created secret to the Job(IncrementalLearningJob).
-```yaml
+
+This image is generated by the script [build_images.sh](/examples/build_image.sh), used for creating training, eval and inference worker.  
+
+### Prepare Job
+* Inference/Train/Eval worker now can be deployed by nodeName and [nodeSelector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) on multiple nodes.  
+
+* make sure following the local dir exist in egde side.    

```

Comment:
better to describe like this:
make sure to follow the local dir which exists in edge side

Reply:
Capital M is missing as well

Reply:
fix